daemon/api_quotas.go

```diff
@@ -176,7 +177,14 @@ func postQuotaGroup(c *Command, r *http.Request, _ *auth.UserState) Response {
 			// then we need to create the quota
 			ts, err = servicestateCreateQuota(st, data.GroupName, data.Parent, data.Snaps, quantity.Size(data.MaxMemory))
 			if err != nil {
-				// XXX: dedicated error type?
+				// check for conflict errors
+				switch err.(type) {
+				case *snapstate.ChangeConflictError:
+					return Conflict(err.Error())
```

Comment:
we want to use errToResponse here or SnapChangeConflict

Reply:
done